deployment/stages/staging.rb

```diff
@@ -13,3 +13,4 @@
 set :keep_releases,  2
 
 before 'deploy:setup', 'sumo:setup:client_folder'
+before 'deploy:setup', 'sumo:db:create'
```

Comment:
Dit moet toch niet in de stage file, maar in de algemene deploy file? Bij prod wil je dit toch ook?